From 594cc7c7d61945a82d10b30d54043c7abda0f9bb Mon Sep 17 00:00:00 2001
From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Fri, 6 Sep 2019 17:47:13 +0800
Subject: [PATCH 17/30] input-settings: Use portable formatting for 64-bit
 integers

On 64-bit systems int64_t is %lx but on 32-bit systems it is %llx.

https://gitlab.gnome.org/GNOME/mutter/merge_requests/778
---
 src/backends/meta-input-settings.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/backends/meta-input-settings.c b/src/backends/meta-input-settings.c
index 3ebf826ac..6a4836d27 100644
--- a/src/backends/meta-input-settings.c
+++ b/src/backends/meta-input-settings.c
@@ -1437,7 +1437,8 @@ lookup_tool_settings (ClutterInputDeviceTool *tool,
     }
   else
     {
-      path = g_strdup_printf ("/org/gnome/desktop/peripherals/stylus/%lx/", serial);
+      path = g_strdup_printf ("/org/gnome/desktop/peripherals/stylus/%" G_GINT64_MODIFIER "x/",
+                              serial);
     }
 
   tool_settings =
-- 
2.23.0

